Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -674,14 +677,15 @@ public String uploadFromHome() {
     public void downloadToHome() {
         /*
          * zunächst prüfen, ob dieser Band gerade von einem anderen Nutzer in
-         * Bearbeitung ist und in dessen Homeverzeichnis abgelegt wurde, ansonsten
-         * Download
+         * Bearbeitung ist und in dessen Homeverzeichnis abgelegt wurde,
```

Comment:
If you change a comment - translate it to English.

Reply:
This is just the code formatter reformatting the comment, I didn't change it.